database/generator.js

```diff
@@ -0,0 +1,84 @@
+const db = require('./');
+const faker = require('faker');
+
+class DummyDataGenerator {
+  constructor() {
+  }
+
+  getRandomStatus() {
+    const status = [
+      1, // want-to-read
+      2, // currently reading
+      3 // read
+    ];
+    return status[Math.floor(Math.random() * status.length)];
+  };
+
+  async createBookInfo() {
```

Comment:
You might want to rename these functions to seedBookInfo() etc..
createBookInfo could be confused with creating mock data while the function ultimately aims to seed the database with user data.

Reply:
Yeah, you are right. I will change the names.